test/ResultSet/ReturnType.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+
```

Comment:
You should remove the extra new lines, they make the build fail but it's also messy